readthedocsext/theme/templates/organizations/organization_form.html

```diff
@@ -16,14 +16,94 @@ <h2 class="ui medium heading">
 
-        <form class="ui form" action="" method="post">
-          {% csrf_token %}
-          {{ form|crispy }}
-
-          <button class="ui primary button" type="submit">
-            {% trans "Get started" %}
-          </button>
-        </form>
+        <div class="ui top attached segment">
+          <div class="ui relaxed list">
+
+            <div class="item">
+              <i class="fad fa-building medium middle aligned icon"></i>
+              <div class="content">
+                <div class="header">
+                  {% trans "My organization is already registered" %}
+                </div>
+
+                <div class="description">
+                  {% blocktrans trimmed %}
+                    Somebody else already created an organization on Read the Docs and told you to sign up.
+                  {% endblocktrans %}
+                </div>
+              </div>
+            </div>
+
+            <div class="item">
+              <i class="fad fa-plus medium middle aligned icon"></i>
+              <div class="content">
+                <div class="header">I want to create a new organization</div>
+                <div class="description">
+                  {% blocktrans trimmed %}
+                    You want to create an organization to start a 30 day trial for your company.
+                  {% endblocktrans %}
+                </div>
+              </div>
+            </div>
+
+            <div class="item">
+              <i class="fad fa-pause medium middle aligned icon"></i>
+              <div class="content">
+                <div class="header">I'm not sure</div>
+                <div class="description">
+                  {% blocktrans trimmed %}
+                    I'm not sure what I'm doing and I need some help.
+                  {% endblocktrans %}
+                </div>
+              </div>
+            </div>
+
+          </div>
+        </div>
+
+        <div class="ui form basic fieldset segment">
+          <div class="field menu">
+            <label class="required">Choose an option</label>
+            <select class="ui fluid dropdown" name="organization" data-bind="semanticui: { tabmenu: {}}">
+              <option class="item" data-tab="registered">My organization is already registered</option>
+              <option class="item" data-tab="new">I want to create a new organization</option>
+              <option class="item" data-tab="not-sure">I'm not sure</option>
+            </select>
+          </div>
+        </div>
+
+        <div class="ui bottom attached active tab" data-tab="registered">
+          <p>
+            {% blocktrans %}
+              We weren't able to find the organization attached to your email domain.
+              Make sure you have registered with your company's email and not your personal email.
+              Contact your organization's owner and tell them to invite you to the organization.
+              They will send you a link with the invitation
```

Comment:
We should make this block more generally applicable to users. For example, none of this really applies to users that are trying to authenticate with GitHub SSO -- we might confuse them telling them to connect with the right email address.

```suggestion
              If you are having trouble joining an existing organization,
              contact one of your organization's owners and request an invite.
              They should email you a link that you can use to join.
```

"Email you a link" describes organization magical links and proper invite emails.